fgrpc/pingsrv.go

```diff
@@ -74,12 +74,14 @@ func PingServer(port, cert, key, healthServiceName string, maxConcurrentStreams
 		log.Infof("Setting grpc.MaxConcurrentStreams server to %d", maxConcurrentStreams)
 		grpcOptions = append(grpcOptions, grpc.MaxConcurrentStreams(maxConcurrentStreams))
 	}
-	if cert != "" && key != "" {
-		creds, err := credentials.NewServerTLSFromFile(cert, key)
+	if tlsOptions.Cert != "" && tlsOptions.Key != "" {
+		tlsCfg, err := tlsOptions.TLSConfig()
 		if err != nil {
 			log.Fatalf("Invalid TLS credentials: %v", err)
 		}
-		log.Infof("Using server certificate %v and key %v to construct TLS credentials", cert, key)
+		creds := credentials.NewTLS(tlsCfg)
+		log.Infof("Using server certificate %v and key %v to construct TLS credentials - mtls %v",
```

Comment:
Not sure if you want to log this?

Reply:
why not? it's just the file names like before + whether mtls is on or not

Reply:
I can add "from" for clarity

Reply:
@wiardvanrij see https://github.com/fortio/fortio/pull/720/commits/38dd76a85b9a2013c5ba96773fccb50c171cae6a - better?

Reply:
Eh maybe I said something too quick. I thought it was the actual key itself that got logged.. all good

Reply:
np, it was ambiguous wording, hopefully clearer/better now (just that the `mutal ` might be hard to spot)